autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/CronTimeEvaluator.java

```diff
@@ -64,6 +68,12 @@ public class CronTimeEvaluator extends EvaluatorExecutor {
 
     private static final Long UPDATE_FAILED_INTERVAL_MINUTES = 60L;
 
+    private static final String YARN = "yarn";
```

Comment:
We cannot be using service names directly. This can change across blueprints - especially custom blueprints. Instead the checks need to be based on the type of service (not the name).

Also - the checks should not be in CronTimeEvaluator - more on this further down.